lib/parser.dart

```diff
@@ -2209,10 +2217,11 @@ class _Parser {
   }
 
   // TODO(terry): Need to handle auto.
-  marginValue(var exprTerm) {
+  num marginValue(var exprTerm) {
```

Comment:
Can we make this parameter a `LiteralTerm`?

Reply:
Done.

Reply:
Why not type it as `Expression` then?

Reply:
Oh sorry, you did! :smile: